assets/store_descriptions/generate_and_validate.py

```diff
@@ -3,20 +3,27 @@
 import xml.etree.ElementTree as ET
 import glob
 import sys
+import os
 
 
 en_tree = ET.parse('assets/store_descriptions/en-US/strings.xml')
 en_root = en_tree.getroot()
 
 def getString(key):
     try:
-        string = root.findall(key)[0].text
+        string = root.findall(".//string[@name='" + key + "']")[0].text
```

Comment:
The use of `root` is ambiguous. I'd recommend to build a class.

Reply:
I'm not sure if I understand you correctly. The var `root` is overwritten each time a new language is loaded. If I create a class I have to save the instance which I will do in a variable. This instance will be lost when a new language is loaded, so I have the exact same thing as before. Right?